README.md

```diff
@@ -49,44 +53,46 @@ If you're self-hosting a Grafana installation you can follow the steps below to
 
 1. Install dependencies
 ```BASH
-# Yarn
 yarn install
-
-# npm
-npm install
 ```
+
 2. Build plugin in production mode
+
 ```BASH
-# Yarn
 yarn build
-
-#npm
-npm run build
 ```
+
 3. Install plugin by copying the files from `./dist` to your Grafana plugin directory
+
 ```BASH
-cp -r ./dist /your/grafana/plugin/directory/k6-cloud
+cp -r ./dist/* /your/grafana/plugin/directory/grafana-k6cloud-datasource/
 ```
+
 **Linux**: By default, the Linux plugin location is: `/var/lib/grafana/plugins`
 
 **macOS**: By default, the Mac plugin location is: `/usr/local/var/lib/grafana/plugins`
 
-4. Make sure you have [configured your Grafana installation](https://grafana.com/docs/grafana/latest/administration/configuration/) to allow unsigned plugins
+4. When building locally, make sure you have [configured your Grafana installation](https://grafana.com/docs/grafana/latest/administration/configuration/#allow_loading_unsigned_plugins) to allow unsigned plugins.
+
 ```INI
 ...
 [plugins]
-allow_loading_unsigned_plugins=true
-...
+allow_loading_unsigned_plugins=grafana-k6cloud-datasource
```

Comment:
Ah, I see I misunderstood how this config setting worked 😅 Thanks for fixing it!